addons/stock/views/stock_picking_views.xml

```diff
@@ -337,6 +337,24 @@
                                                 ('parent.show_reserved', '=', False),
                                                 ('parent.show_operations', '=', False)]}"/>
                                 </tree>
+                                <form string="Stock Moves">
+                                    <header>
+                                        <field name="state" widget="statusbar"/>
+                                    </header>
+                                    <group>
+                                        <field name="product_uom_category_id" invisible="1"/>
```

Comment:
needed?

Reply:
```product_uom_category_id``` is used in domain of ```product_uom``` field that's why added here and keep it invisible